file_paths.md

```diff
@@ -252,6 +253,7 @@ create a `puppet` user or group.
                 pxp-agent.log                 # enabled by default
                 pxp-access.log                # not enabled by default
             spool *                           # directory containing results of pxp-agent modules
+            tasks-cache *                     # directory containing a cache of files for all downloaded tasks
```

Comment:
Yes. On Windows, the tasks-cache wasn't in the var directory, it was at pxp-agent/tasks-cache.